assets/js/storage.js

```diff
@@ -37,19 +39,25 @@ angular
           likes.push(video);
         }
         setObj('likes', likes);
+        likedVideos = likes;
       } else {
         setObj('likes', [video]);
+        likedVideos = likes;
       }
     },
 
     likesVideo(videoKey) {
-      let likes = getObj('likes');
+      let likes = likedVideos;
 
       if (likes) {
         return !!likes.find((l) => l.key === videoKey);
       } else {
         return false;
       }
+    },
+
+    getLikedVideos() {
+      return likedVideos;
```

Comment:
This line threw lots of errors when directly calling `getObj('likes')` because the resulting array was a completely new array.